downloads/swan-lake-release-notes/swan-lake-2201.1.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,688 @@
+
+#### `os` package
+- Introduced the `setEnv()` function to set an environment variable
+- Introduced the `unsetEnv()` function to remove an environment variable from the system
+- Introduced the `listEnv()` function to list the existing environment variables of the system
+
+### Improvements
+
+#### `http` package
+
+- Allowed `Caller` to respond to an `error` or a `StatusCodeResponse`
+- Appended the HTTPS scheme (`https://`) to the client URL if security is enabled
+- Refactored the auth-desugar response with a `DefaultErrorInterceptor`
+- Hid the subtypes of the `http:Client`
+
+#### `jwt` package
+
+- Appended the HTTPS scheme (`https://`) to the client URL (of JWKs endpoint) if security is enabled
+
+#### `oauth2` package
+
+- Appended the HTTPS scheme (`https://`) to the client URL (of token endpoint or introspection endpoint) if security is enabled
+
+### Bug fixes
+
+#### `grpc` package
+
+- Fixed incorrect stub generation for repeated values of any, struct, timestamp, and duration messages
+- Fixed incorrect caller type name validation in the gRPC compiler plugin
+- Fixed passing Protobuf predefined types as repeated values and values in messages
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.1.0%22+label%3AType%2FBug).
+
+## Deployment updates
+
+### New features
+
+Added the `name` field for the `cloud.config.files` property in the `Cloud.toml` file to change the name of the generated config map 
+
+### Improvements
+- Reduced the package size of `ballerina/cloud`
+- Docker image generation now relies on the user's docker client
+- Added the `ballerinax/awslambda` package in [Ballerina Central](https://central.ballerina.io/ballerinax/awslambda)
+- Added the `ballerinax/azure_functions` package in [Ballerina Central](https://central.ballerina.io/ballerinax/azure.functions)
+
+### Breaking changes
+
+For existing `ballerinax/awslambda` and `ballerinax/azure_functions` projects, change the version to `2.1.0` in the `Dependencies.toml` file.
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Ballerina+Swan+Lake+-+2201.1.0%22).
+
+## Developer tools updates
+
+### New features
+
+#### AsyncAPI tool
+
+Introduced the Ballerina AsyncAPI tool, which will make it easy for you to start the development of an event API documented in an AsyncAPI contract in Ballerina by generating Ballerina service and listener skeletons. Ballerina Swan Lake supports the AsyncAPI Specification version 2.x. For more information, see [Ballerina AsyncAPI support](/learn/ballerina-asyncapi-support) and [AsyncAPI CLI documentation](/learn/cli-documentation/asyncapi/#asyncapi-to-ballerina).
```

Comment:
Provide the link to AsyncAPI Specification version 2.x.

Reply:
Seems like a common link is not available pointing all the versions. But maybe we can go with https://www.asyncapi.com/docs/specifications/v2.0.0. It points to the rest of the specifications.

Reply:
@HasithaAthukorala

Reply:
Added via https://github.com/ballerina-platform/ballerina-dev-website/pull/4402/commits/ca8f32cdbad002463f4a82a020b3aed1308454f7.